### ISSUE TITLE
Instant missed calls, 100% repeatable

### ISSUE THREAD

**Issue by MrPeteH** [signalapp/Signal-Android#12901]
- [ x] I have searched open and closed issues for duplicates
- [x ] I am submitting a bug report for existing functionality that does not work as intended
- [ x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [ x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Sitting with two tech friends. Two of us have identical phones. 
I can call them, no prob. 
If either calls me, voice or video, then:
* they hear ringing
* I immediately see "missed (voice|video) call" 

I read through bug reports... 

### Steps to reproduce
So far, I can't make this NOT happen on my phone for 24+ hours in several contexts. 
* all perms enabled
* nothing blocked
* not DND
* they can send me Signal chat msg OK. Has no impact. 

I notice an interesting yellow highlighted item in the log that seems to relate... Will fill in detail on that after I submit this. 

### Device info

**Device:** Samsung Note10+
**Android version:** 12
**Signal version:** 6.17.3

### Link to debug log

https://debuglogs.org/android/6.17.3/e47e9321c43d2cafedc24c7a6c16930d75d0f67873e3b40291ed682d84f6aa45

**Comment by greyson-signal** [comment]
Hi there, it seems like you have a notification profile enabled that ignores calls? You can check via Settings > Notifications > Notification profiles (or directly through three-dot-menu > Notification profile)

**Comment by MrPeteH** [comment]
Wow.
A) you are correct!
B) I have that, yet have no recollection of setting it up, certainly not with intent to block all but those in the group!

That profile is "urgent" on my app... And yes "allow all calls" is off...

**Comment by benschac** [comment]
I am able to reproduce this on a pixel 6a today.  Very much still any issue

**Comment by greyson-signal** [comment]
@benschac What do you mean by "reproduce"? The initial issue was a user-configuration problem where someone had a notification profile that blocked calls that they setup that way by accident. Are you having that same issue or do you not have a notification profile setup?

**Comment by benschac** [comment]
That's the default configuration, which is quite cumbersome. On iOS I receive a call notif by default, so I don't miss calls. On android, I have to do this whole configuration dance which is the issue.


I guess it's more UX than regression or defect. Either way, it was tricky finding that config. I had to search github to find this an update the configuration which was a poor user experience IMO.

**Comment by benschac** [comment]
Tytytyty ❤️❤️

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by MrPeteH** [comment]
I just played with the UI. Boy is this non-obvious. (Yes, it does say up front "Create a profile to receive not'ns only from people/groups you choose.")

I think one VERY simple edit would help make it more obvious: rename the sample profile from "Urgent" to "Urgent only" 

But then... here's a strange challenge: that settings area has options like Exceptions: Allow all calls, and Notify for all mentions.
NONE of that is documented anywhere I can find online. Can you?
I can't even find a description of this feature.

For example:
* If exceptions are "allow all calls" and "notify for all mentions"... then what are the other notifications?
* In other words, what's this list of people supposed to accomplish?
;)

**Comment by szotsaki** [comment]
Probably a duplicate of https://github.com/signalapp/Signal-Android/issues/7733
